### PR TITLE
Wallet UI: Improve views

### PR DESCRIPTION
**Pull request by dennisreimann**
Again, some improvements for the wallet views. Mostly properly aligned things, set the correct widths for inputs and minor text updates. 

## Derivation scheme

Made the input a textarea, so that the full scheme is visible at once. Also removed this part of the text, that no one seems to remember what it was for :D

> Please, verify that you are generating the right addresses by clicking on 'Check ExtPubKey'

derivation-scheme

## Send 

Mostly alignment stuff and responsive improvements here …

send-plain

send-coin-selection

## Receive

Mostly alignment stuff and responsive improvements here …

receive

## Rescan

Input sizes and responsive improvements.

rescan

## Settings

Input sizes and responsive improvements.

settings

### CODE REVIEW

File: BTCPayServer/Views/Wallets/WalletSend.cshtml [hunk 1]
```diff
@@ -233,10 +230,9 @@
     <style>
         .remove-destination-btn{
             font-size: 1.5rem;
-            border-radius: 0;
         }
-         .remove-destination-btn:hover{
-                   background-color: #CCCCCC;
-         }
+     .remove-destination-btn:hover{
```
Comment (bolatovumar) [inline BTCPayServer/Views/Wallets/WalletSend.cshtml:234]: Seems like alignment with the above code got out of whack here?

File: BTCPayServer/Views/Wallets/WalletSettings.cshtml [hunk 2]
```diff
@@ -45,35 +36,38 @@
             }
             @for (int i = 0; i < Model.AccountKeys.Count; i++)
             {
-                <hr/>
-                <h5>Account key @i</h5>
+                <hr class="my-4"/>
+
+                <h4 class="mb-3">Account key @i</h4>
```
Comment (bolatovumar) [inline BTCPayServer/Views/Wallets/WalletSettings.cshtml:41]: Should this stay an `h5` since we have an `h4` above for this whole section?

Comment (dennisreimann) [reply BTCPayServer/Views/Wallets/WalletSettings.cshtml:41]: I consider this information on the same level and visually it seemed better this way, but I'm open for vetos :)

File: BTCPayServer/Views/Wallets/WalletSend.cshtml (hunk 1 above)
Comment (dennisreimann) [reply BTCPayServer/Views/Wallets/WalletSend.cshtml]: Thanks for the pointer, fixed it. 👍

File: BTCPayServer/Views/Wallets/WalletSettings.cshtml (hunk 2 above)
Comment (bolatovumar) [reply BTCPayServer/Views/Wallets/WalletSettings.cshtml:41]: Yeah, it's fine. It's not a big deal.

**Comment by NicolasDorier** [comment]
I want to merge!

**Comment by dennisreimann** [comment]
I want @kukks to have a look first, just to ensure I haven't forgotten something in the send dialogs.

**Comment by dennisreimann** [comment]
@Kukks can you take a look and verify?

File: BTCPayServer/Views/Stores/AddDerivationScheme.cshtml [hunk 3]
```diff
@@ -72,12 +72,15 @@
...
+                    <h5>Derivation scheme</h5>
+                    <p>
+                        The derivation scheme represents the destination of the funds received by your invoice.
+                        It is generated by your wallet software.
+                    </p>
                 </div>
                 <div class="form-group">
                     <label asp-for="DerivationScheme"></label>
-                    <input asp-for="DerivationScheme" class="form-control store-derivation-scheme" />
+                    <textarea asp-for="DerivationScheme" class="form-control store-derivation-scheme text-monospace py-2" rows="2"></textarea>
```
Comment (Kukks) [inline BTCPayServer/Views/Stores/AddDerivationScheme.cshtml:83]: This change didnt break any tests?

File: BTCPayServer/Views/Stores/AddDerivationSchemes_HardwareWalletDialogs.cshtml [hunk 4]
```diff
@@ -9,29 +9,51 @@
...
+                    <thead>
+                        <tr>
+                            <th>Wallet</th>
+                            <th>Instructions</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>Cobo Vault</td>
+                            <td><kbd>Settings ❯ Watch-Only Wallet ❯ Wasabi Wallet/BTCPay ❯ Export Wallet</kbd></td>
```
Comment (Kukks) [inline BTCPayServer/Views/Stores/AddDerivationSchemes_HardwareWalletDialogs.cshtml:31]:
```suggestion
                            <td><kbd>Settings ❯ Watch-Only Wallet ❯ BTCPay ❯ Export Wallet</kbd></td>
```

File: BTCPayServer/Views/Stores/AddDerivationSchemes_HardwareWalletDialogs.cshtml [hunk 5]
```diff
@@ -9,29 +9,51 @@
...
+                        <tr>
+                            <td>ColdCard</td>
+                            <td>
+                                <kbd>Advanced ❯ MicroSD Card ❯ Electrum Wallet</kbd> or
+                                <kbd>Advanced ❯ MicroSD Card ❯ Wasabi Wallet</kbd>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Electrum</td>
+                            <td><kbd>Filed ❯ Save Copy</kbd></td>
```
Comment (Kukks) [inline BTCPayServer/Views/Stores/AddDerivationSchemes_HardwareWalletDialogs.cshtml:42]:
```suggestion
                            <td><kbd>File ❯ Save Copy</kbd></td>
```

File: BTCPayServer/Views/Wallets/WalletPSBT.cshtml [hunk 6]
```diff
@@ -51,9 +51,11 @@
             </div>
             <pre><code class="json">@Model.Decoded</code></pre>
         }
-        <h3>PSBT to decode</h3>
+        <h4 class="mb-3">Decode PSBT</h4>
+        <p>You can decode the PSBT by either pasting its content or by uploading the file.</p>
```
Comment (Kukks) [inline BTCPayServer/Views/Wallets/WalletPSBT.cshtml:55]:
```suggestion
        <p>You can decode a PSBT by either pasting its content or by uploading the file.</p>
```

File: BTCPayServer/Views/Stores/AddDerivationScheme.cshtml (hunk 3 above)
Comment (dennisreimann) [reply BTCPayServer/Views/Stores/AddDerivationScheme.cshtml:83]: Why did you expect it to? I think from Seleniums perspective it's just a bigger input field, so as long as we aren't specifically selecting `input` we should be good.

**Comment by Kukks** [comment]
Just tiny nits not even related to PR but since you adjusted the same area might as well include them.

**Pull request merged**